app/views/early_career_payments/claims/eligibility_confirmed.html.erb

```diff
@@ -1,48 +1,25 @@
-<% content_for(:page_title, page_title("You are eligible to claim for early-career payment", policy: current_policy_routing_name)) %>
+<% content_for(:page_title, page_title("You’re eligible for an additional payment", policy: current_policy_routing_name, show_error: current_claim.errors.any?)) %>
```

Comment:
It's not necessary with UTF-8 encoding: see https://www.gov.uk/government/publications/open-standards-for-government/cross-platform-character-encoding-profile.

Reply:
We put in the HTML entities in GHWT because curly quotes can be so difficult to spot. Curly quotes are a pain generally if you try to search for text like "you're" but you've typed a straight quote (like you would naturally) then you don't get a match.

Reply:
Basically if we can get away with straight quotes, it's less of a headache all round

Reply:
Agreed (I had to re-type it because of the quote) and I'm not sure what the DfE convention is, but the content designer is responsible for this. The original content I replaced had curly quotes as well.

Reply:
It's worth checking with them, because they might have copy-and-pasted themselves from a "helpful" app like Microsoft Word or Apple Mac Notes which automatically converts to curly quotes. If the designers haven't intentionally asked for curly quotes I'd change them to straight quotes. Also, if they don't understand what you're even asking, I'd just change them to straight quotes.

Reply:
VS Code highlights these quotes, so they are probably not the recommended way.

Reply:
Confirmed we should use curly apostrophes.